migtests/scripts/resumption.sh

```diff
@@ -89,8 +105,18 @@ main() {
 	  ./generate_config.py
 	fi
 
-	step "Run import with resumptions"
+	# Backup original config
+	cp config.yaml config.yaml.original
 
+	# Append additional config to config.yaml if specified
```

Comment:
where is this config.yaml reside?
Is it being created at run by the script

Reply:
lets connect tomorrow once, need to understand the capabilities of the resumption script.